src/sentry/tasks/assemble.py

```diff
@@ -251,53 +251,68 @@ def _normalize_debug_id(debug_id: Optional[str]) -> Optional[str]:
         return None
 
 
-def _extract_debug_ids_from_manifest(manifest: dict) -> List[Tuple[SourceFileType, str]]:
+def _extract_debug_ids_from_manifest(
+    manifest: dict,
+) -> Tuple[str, List[Tuple[SourceFileType, str]]]:
```

Comment:
```suggestion
) -> Tuple[Optional[str], List[Tuple[SourceFileType, str]]]:
```

As the `bundle_id` could be missing as well?

Reply:
Thanks! Yes